### ISSUE TITLE
cart shortcode, load specific product from cart

### ISSUE THREAD

**Issue by johnbailon** [jigoshop/jigoshop#642]
Trying to develop a plugin *(for the first time)* that allows for a dynamically generated product. I'd like to request this change:
on `shortcodes/cart.php` line 118 (and also 131) contains this filter:

From:
```php
echo esc_url( apply_filters('jigoshop_product_url_display_in_cart', get_permalink($values['product_id']), $values['product_id']) );
```

To
```php
echo esc_url( apply_filters('jigoshop_product_url_display_in_cart', get_permalink($values['product_id']), $values) );
```
Changing the 2nd argument to `$values` instead of `$values['product_id']`. This way, I can go back to that specific product and variation (with my custom `$cart_item_data`) which I can then load onto `single-product.php` with a new custom action. Then updating the cart with that specific cart_item is next.

Bottom line, I want to be able to go back to a specific cart_item's single-product view to edit/change it. As far as I've looked, that's not currently possible. Any thoughts?
